lib/roadie/rails.rb

```diff
@@ -3,6 +3,8 @@ module Rails
   end
 end
 
+require "active_support/concern"
```

Comment:
I would prefer not to use `ActiveSupport::Concern`, but I cannot find any rational reason not to (we're inside the `Rails` adapter, after all!) so I'll approve it.